lib/jekyll-remote-theme/downloader.rb

```diff
@@ -7,8 +7,13 @@ class Downloader
 
       HOST = "https://codeload.github.com".freeze
       PROJECT_URL = "https://github.com/benbalter/jekyll-remote-theme".freeze
-      USER_AGENT  = "Jekyll Remote Theme/#{VERSION} (+#{PROJECT_URL})".freeze
       TEMP_PREFIX = "jekyll-remote-theme-".freeze
+      TYPHOEUS_OPTIONS = {
+        :headers => {
+          :user_agent => "Jekyll Remote Theme/#{VERSION} (+#{PROJECT_URL})",
+        },
+        :verbose => (Jekyll.logger.level == :verbose),
```

Comment:
This should be :debug instead of :verbose for the jekyll log level.

Reply:
oooph! yes.